tensorflow/buildspec-tf1.yml

```diff
@@ -84,23 +84,23 @@
         docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
         context:
           <<: *TRAINING_CONTEXT
-      BuildTensorflowCPUInferencePy3DockerImage:
+      BuildTensorflowCPUInferencePy36DockerImage:
         <<: *INFERENCE_REPOSITORY
-        build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+        build: &TENSORFLOW_CPU_INFERENCE_PY36 false
```

Comment:
We need to retain original code for py3 buildspec logic in addition to py36 logic. Otherwise, It is creating only py2 and py36 image, not py3 image.